### PR TITLE
MNT Use shared travis config, use sminnee/phpunit

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,73 +1,4 @@
...
-  - if [[ $NPM_TEST ]]; then nvm install $TRAVIS_NODE_VERSION && nvm use $TRAVIS_NODE_VERSION && npm install -g yarn && yarn install --network-concurrency 1 && (cd vendor/silverstripe/admin && yarn install --network-concurrency 1) && yarn run build; fi
-
-script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
-  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src tests; fi
-  - if [[ $NPM_TEST ]]; then git diff-files --quiet -w --relative=client; fi
-  - if [[ $NPM_TEST ]]; then git diff --name-status --relative=client; fi
-  - if [[ $NPM_TEST ]]; then yarn run coverage; fi
-  - if [[ $NPM_TEST ]]; then yarn run lint; fi
-  - if [[ $BEHAT_TEST ]]; then vendor/bin/behat @security-extensions; fi
```
Comment (dnsl48) [inline .travis.yml:65]: I'm wondering, did we lose these behat tests (`@security-extensions` suite) somewhere along the road? Might be worth investigating before we get rid of it completely.

**Review by dnsl48 — CHANGES_REQUESTED** [review]
Need to make sure the behat suite is a ghost of the past before we remove it completely

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml:65]: I've looked through previous releases https://github.com/silverstripe/silverstripe-security-extensions/releases - I don't think these behat tests ever existed.  I suspect there was an intention to write some at some stage, though they were never actually written

Comment (dnsl48) [reply .travis.yml:65]: ok, let's ship it then

**Review by dnsl48 — APPROVED** [review]

**Pull request merged**